cmd/kube-proxy/proxy.go

```diff
@@ -52,6 +53,7 @@ func init() {
 }
 
 func main() {
+	util.AddAllFlagsToPFlags()
```

Comment:
Add added a new file and `util.InitFlags`.   There are places where we do some other stuff between initing flags and logs and so I'm going to leave it for now.